src/checks/lowercase_key.rs

```diff
@@ -0,0 +1,51 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct LowercaseKeyChecker {}
+
+impl Check for LowercaseKeyChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        let line_str: Vec<&str> = line.raw_string.split('=').collect();
+        let key = line_str[0];
+        if key.to_uppercase() == key {
+            None
+        } else {
+            Some(Warning {
+                message: format!("The {} key should be in uppercase", key),
+            })
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn lowercase_key_checker_run() {
+        let checker = LowercaseKeyChecker {};
```

Comment:
Please, use the `trait Default`:
```suggestion
        let checker = LowercaseKeyChecker::default();
```